scripts/test.js

```diff
@@ -12,11 +12,34 @@
 // which produces invalid config.
 // See: https://github.com/ckeditor/ckeditor5-react/issues/25
 
-const getKarmaConfig = require( './utils/getkarmaconfig' );
-const { Server: KarmaServer } = require( 'karma' );
+const options = parseArguments( process.argv.slice( 2 ) );
 
-const config = getKarmaConfig();
+const { Server: KarmaServer } = require( 'karma' );
+const getKarmaConfig = require( './utils/getkarmaconfig' );
+const server = new KarmaServer( getKarmaConfig() );
 
-const server = new KarmaServer( config );
+console.log( `
+╓───┄                   ┄───╖
+║ Starting tests for Vue v${ options.vue } ║
+╙───┄                   ┄───╜
+` );
 
 server.start();
+
+/**
+ * Parses CLI arguments and options.
+ *
+ * @param {Array.<String>} args CLI arguments and options.
+ * @returns {Object} options
+ * @returns {Number} options.vue Desired Vue version for test run.
+ */
+function parseArguments( args ) {
+	return require( 'minimist' )( args, {
+		alias: {
+			v: 'vue'
+		},
+		default: {
+			vue: 2
```

Comment:
Let's rename `vue` argument to `vue-version`?

Reply:
Done.